pyiron_atomistics/atomistics/structure/factories/aimsgb.py

```diff
@@ -2,8 +2,10 @@
 # Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
 # Distributed under the terms of "New BSD License", see the LICENSE file.
 
-from structuretoolkit.build import grainboundary, get_grainboundary_info
```

Comment:
Please use the structure toolkit interface.

Reply:
The interface is pretty similar and this is what is used by `pyiron_atomistics` in the background https://github.com/pyiron/structuretoolkit/blob/main/structuretoolkit/build/aimsgb.py

Reply:
I opened a jupyter notebook from the jupyterhub session runnig at cmti and tried to import `from structuretoolkit.common.pymatgen import ase_to_pymatgen, pymatgen_to_ase` and got this error `ModuleNotFoundError: No module named 'structuretoolkit.common'`

`from structuretoolkit.build import grainboundary, get_grainboundary_info` also gave me the error `ImportError: cannot import name 'grainboundary' from 'structuretoolkit.build' (/u/system/SLES12/soft/pyiron/dev/anaconda3/lib/python3.10/site-packages/structuretoolkit/build/__init__.py)`

Reply:
Oh, I got this. I have to modify the import statement from `from structuretoolkit.build import grainboundary, get_grainboundary_info` to `from structuretoolkit.build.aimsgb import grainboundary_info, grainboundary_build`

But, still can't figure out what is the new alternative of `structuretoolkit.common.pymatgen import ase_to_pymatgen, pymatgen_to_ase`

Reply:
Now I am confused, because it is available in the official package https://github.com/pyiron/structuretoolkit/blob/main/structuretoolkit/common/pymatgen.py . Which version of structure toolkit is installed on cmti? You can check the version using `conda list structuretoolkit`

Reply:
The output of `conda list structuretoolkit` is as follows:

`# packages in environment at /u/system/SLES12/soft/pyiron/dev/anaconda3:
#
# Name                    Version                   Build  Channel
structuretoolkit          0.0.1              pyhd8ed1ab_1    conda-forge

Note: you may need to restart the kernel to use updated packages.`

Reply:
# packages in environment at /u/system/SLES12/soft/pyiron/dev/anaconda3:
#
# Name                    Version                   Build  Channel
structuretoolkit          0.0.1              pyhd8ed1ab_1    conda-forge

Note: you may need to restart the kernel to use updated packages.

Reply:
Ok, this is a rather old version - the current version is `0.0.7`.

Reply:
@jan-janssen Thanks! I noticed that the current incompatibility issue is coming from this line `AseAtomsAdaptor.get_structure(initial_struct, cls=None)`. The option `cls=None` is no longer accepted. We can change it to `AseAtomsAdaptor.get_structure(initial_struct)` which will take the default value which is `pymatgen.Structure`